movo_robot/movo_bringup/launch/manipulation/movo_moveit.launch

```diff
@@ -11,7 +11,9 @@
     <include if="$(optenv MOVO_HAS_KINOVA_ARM_7DOF false)" file="$(find movo_7dof_moveit_config)/launch/movo_moveit_planning_execution.launch"/>
     <include if="$(optenv MOVO_HAS_KINOVA_ARM_6DOF false)" file="$(find movo_moveit_config)/launch/movo_moveit_planning_execution.launch"/>
     
-	<node pkg="movo_bringup" type="init_robot" name="init_robot" respawn="false" output="screen">
+	<!-- Removed arm init since motion is not necessary and impedes box urdf integration-->
+	<!--node pkg="movo_bringup" type="init_robot" name="init_robot" respawn="false" output="screen"-->
+
     	<param if="$(optenv MOVO_HAS_KINOVA_ARM_7DOF false)" name="jaco_dof" value="7dof" />
```

Comment:
I was not sure if these config file loading are useful anywhere else so I've left them in.